docs/REST API/rest-api.md

```diff
@@ -7404,21 +7404,60 @@ This allows you to easily configure scheduled queries that will impact a whole t
 ## Scripts
 
 - [Run script](#run-script)
-- [Get script result](#get-script-result)
-- [Run live script](#run-live-script)
 - [Add script](#add-script)
-- [Delete script](#delete-script)
+- [Get script result](#get-script-result)
+- [Upload script](#upload-script)
+- [Download script](#download-script)
```

Comment:
let's keep the "get or download" name for this

Reply:
I guess all I was trying to do there was make upload & download parallel. I assume that the reason some of the descriptions have "get" is to reflect the http request. If you have a convention, I think that's fine but it doens't seem consistently applied.